python/lsst/ts/atspectrograph/atspec_csc.py

```diff
@@ -832,6 +862,13 @@ async def configure(self, config):
                 for key in gratings_str:
                     gratings_str[key] += ","
 
+        # Backward compatibility with xml < 7.2
```

Comment:
Don't understand this but ok...
what should it be replaced with once 7.2 is deployed?

Reply:
nothing... it is a change intended for xml 7.2 but to keep the code backward compatible with xml 7.1.